src/symbol_translations.jl

```diff
@@ -0,0 +1,121 @@
+                                            :atan => "\\arctan",
+                                            :atan2 => "\\arctan",
+                                            :asinh => "\\mathrm{arcsinh}",
+                                            :sinc => "\\mathrm{sinc}",
+                                            :acosh => "\\mathrm{arccosh}",
+                                            :cosc => "\\mathrm{cosc}",
+                                            :atanh => "\\mathrm{arctanh}",
+                                            :acot => "\\mathrm{arccot}",
+                                            :acoth => "\\mathrm{arccoth}",
+                                            :asec => "\\mathrm{arcsec}",
+                                            :sech => "\\mathrm{sech}",
+                                            :asech => "\\mathrm{arcsech}",
+                                            :acsc => "\\mathrm{arccsc}",
+                                            :csch => "\\mathrm{csch}",
+                                            :acsch => "\\mathrm{arccsch}",
+                                            ## Misc
+                                            :log => "\\log",
+                                            :log10 => "\\log_{10}",
+                                            :log2 => "\\log_{2}",
+                                            :gamma => "\\Gamma", # The Gamma function
+                                           )
+
+const trigonometric_functions = [
+                                 :sin,
+                                 :cos,
+                                 :tan,
+                                 :cot,
+                                 :sec,
+                                 :csc,
+                                 :sinh,
+                                 :cosh,
+                                 :tanh,
+                                 :coth,
+                                 :asin,
+                                 :acos,
+                                 :atan,
+                                 :atan2,
+                                 :asinh,
+                                 :sinc,
+                                 :acosh,
+                                 :cosc,
+                                 :atanh,
+                                 :acot,
+                                 :acoth,
+                                 :asec,
+                                 :sech,
+                                 :asech,
+                                 :acsc,
+                                 :csch,
+                                 :acsch,
+                                ]
+
+const comparison_operators = Dict(
+                            :< => "<",
+                            :.< => "<",
+                            :> => ">",
+                            :.> => ">",
+                            Symbol("==") => "=",
+                            Symbol(".==") => "=",
```

Comment:
We don't need `Symbol` here.
```suggestion
                            :(==) => "=",
                            :.== => "=",
```

```julia
julia> :(==) === Symbol("==")
true

julia> :.== === Symbol(".==")
true
```